### DISCUSSION TITLE
Consistent argument name for a model output tbl

### DISCUSSION THREAD

**Discussion by elray1** [hubverse-org/.github#15] [General]
We have 4 different packages with 4 different names for this argument:

- `hubData::coerce_to_hub_schema`: uses [tbl](https://github.com/Infectious-Disease-Modeling-Hubs/hubData/blob/059aa13fd0f97b31ef52dca396b297385491149a/R/coerce_to_hub_schema.R#L3)
- `hubVis::plot_step_ahead_model_output`: uses [model_output_data](https://github.com/Infectious-Disease-Modeling-Hubs/hubVis/blob/fc72622b9852380d0e5c5b184710da57d4f152bc/R/plot_step_ahead_model_output.R#L5)
- `hubEnsembles::simple_ensemble`: uses [model_outputs](https://github.com/Infectious-Disease-Modeling-Hubs/hubEnsembles/blob/105308f174a834765bab93e2040744d916ff607b/R/simple_ensemble.R#L5)
- `hubEvals::transform_point_model_out`: in progress, not yet committed, uses `model_out_tbl`.

Let's pick one name for this argument and use it in all of our packages

**Comment by annakrystalli** [comment]
Just a note that the `hubData` argument does not belong on this list because it refers to submission data rather than data that has been successfuly assimilated into a hub. For example, it never contains a `model_id` column which `model_out_df`s always do.

**Comment by LucieContamin** [comment]
It was decided in dev meeting to standardized argument name for model output tbls to `model_out_tbl`. I will open issue in the 3 packages concern by that.
